### PR TITLE
zzzy: change distfile

### PR DESCRIPTION
**Pull request by meator**
I wan't able to find any alternative source for `zzzy`. PopCorn says that one guy uses it.

Ping @daniel-eys (maintainer)

#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by classabbyamp** [comment]
https://github.com/jgarte/zzzy

**Comment by meator** [comment]
Well I guess I can't search. Thanks @classabbyamp!

File: srcpkgs/zzzy/template [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # Template file for 'zzzy'
 pkgname=zzzy
 version=1
-revision=1
+revision=2
```
Comment (paper42) [inline srcpkgs/zzzy/template:4]: you don't have to increase revision if nothing really changed

Comment (meator) [reply srcpkgs/zzzy/template:4]: I changed the package metadata (`homepage`). Isn't this a reason for revbumping?

Comment (paper42) [reply srcpkgs/zzzy/template:4]: That's hard to say, technically yes, but not sure if it's worth the resources, noone really looks at homepage anyway.

**Pull request merged**